cover-wd.tex

```diff
@@ -22,11 +22,8 @@
 \vspace{2.5cm}
 \begin{center}
 \textbf{\Huge
-\Cpp Extensions for Ranges, \\
-Speculative Combined Proposal Document \\
-(now with Proxy Iterators!)\footnote{This document combines the text of the
-current Ranges TS working draft N4560 with the wording of proposals P0022,
-P0370, and P0441 for review purposes.}}
+\Cpp Extensions for Ranges
+}
 \end{center}
 \vfill
 \textbf{Note: this is an early draft. It's known to be incomplet and
```

Comment:
Looking forward to the time when we get to delete this "incomplet and incorrekt" disclaimer.